packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -235,12 +235,16 @@ export async function wrapRunMigration(context: MigrationContext, exec: Migratio
       if (!(error instanceof Error)) {
         error = new Error(error.toString());
       }
+      let errorMessage = error.message;
+      if (context.currentStep) {
+        errorMessage = `MigrationStep: ${context.currentStep}\n${errorMessage}`;
+      }
```

Comment:
what about: 
const errorMessage = context.currentStep ? `MigrationStep: ${context.currentStep}\n${error.message}` : error.message;

since const will be better;

Reply:
Thanks. Fixed.